webapp/admin/audio_ingest.py

```
@@ -0,0 +1,562 @@
+                            'type': 'pulse',
+                            'device_id': str(idx),
+                            'device_index': idx,
+                            'name': device_info['name'],
+                            'description': f"PulseAudio: {device_info['name']}",
+                            'sample_rate': int(device_info['defaultSampleRate']),
+                            'max_channels': device_info['maxInputChannels'],
+                        })
+                pa.terminate()
+            except ImportError:
+                logger.debug('pyaudio not available for device discovery')
+            except Exception as exc:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)